probe/main.go

```diff
@@ -153,3 +176,16 @@ func interrupt() chan os.Signal {
 	signal.Notify(c, syscall.SIGINT, syscall.SIGTERM)
 	return c
 }
+
+func interfaces() string {
```

Comment:
just taking eth* will missing inter-container traffic on this host.  Your probably better of just taking docker0 and weave bridges.  I'd talk to someone who knows more though.